pyproject.toml

```diff
@@ -1,3 +1,38 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.2"]
+
+[project]
+authors = [{name = "Matias Aguirre", email = "matiasaguirre@gmail.com"}]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Topic :: Internet"
+]
+dependencies = ["social-auth-core >= 4.1.0"]
```

Comment:
Can we remove the requirements.txt? If you think it's too big change to do in this PR, please create another PR on top of this so that we can see a full scope of changes this will bring. Thanks.

Reply:
IMO first remove `dependencies` from setup.py as a separate PR and then come back to this.